packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -59,5 +65,7 @@ auth.protect = async (...args: any[]) => {
     notFound,
     redirect,
   });
+
+  // @ts-ignore
```

Comment:
Can we include the reasoning for ts-ignore here?

Reply:
i was meant to, i forgot 😄